### ISSUE TITLE
Black Screen after sending the kernel via imx usb

### ISSUE THREAD

**Issue by dankzegriefer** [fail0verflow/switch-linux#3]
On three different computers I attempted to upload the kernel and while Fusée Gelée works, with this, I can upload coreboot and then uboot, and uboot is detected by lsusb, the kernel after being sent still gives a black screen as if the LCD isn't started. 
[Image.gz](https://github.com/fail0verflow/switch-linux/files/1954757/Image.gz)
Image.gz is my kernel build.

**Comment by SwtcR** [comment]
We can't do anything without UART output to see what went wrong. You need to build a 1.8V UART cable to connect to the joy-con port.

**Comment by dankzegriefer** [comment]
I have an Arduino, I remember someone I know making it a UART reader but someone told me the voltage is too high for the Switch, is this true?

**Comment by q3k** [comment]
Yes, you will need a voltage converter (like a TXS0108E) or a 1v8 compatible USB/UART adapter.

**Comment by dankzegriefer** [comment]
I found the issue was my SD. New SD fixed it.

**Comment by tardyp** [comment]
Black screen after boot is usually because of a panic. panic is usually because of bad rootfs or bad sdcard

Upon panic, the switch automatically reboot into RCM because of fb62bd9f4f2b. 

Those without serial probably want to revert that config, so that they can more easily stare at there screens.
